website/content/docs/connect/gateways/api-gateway/configuration/api-gateway.mdx

```diff
@@ -0,0 +1,273 @@
+---
+layout: docs
+page_title: API Gateway Configuration Entry Reference
+description: Learn how to configure a Consul API Gateway on VMs.
```

Comment:
Also, wondering if we want to do something like "Learn how to Configure an API Gateway" since the "Consul" bit is kind of superfluous. Also, the native stuff is pretty much how anything outside of Kubernetes ecosystems is going to run -- i.e. Nomad, ECS, etc.

Reply:
I understand the concerns, but as I understand it, these are the main fields that the search engine uses to surface content. My thinking was that, at least for now, people might include "VMs" as part of their search criteria. I wouldn't expect real users to search that way -- more like a few internal people who would expect to see something like "API gateway on VMs" turn up in a search. I agree with you, but it might be something we'll to address, anyway, when these are THE APIGW docs.  As far as repetitiveness, yeah, but it's on purpose so that we can cast a wide net. We also have a metadata template that we use